frontend/src/framework/http-api/base-http-api.package.ts

```diff
@@ -82,8 +82,12 @@ class BaseHttpApi implements HttpApi {
 
         if (hasAuth) {
             const token = await this.storage.get<string>(StorageKey.TOKEN);
-
-            headers.append(HttpHeader.AUTHORIZATION, `Bearer ${token ?? ''}`);
+            if (token) {
+                headers.append(
+                    HttpHeader.AUTHORIZATION,
+                    `Bearer ${token ?? ''}`,
+                );
+            }
```

Comment:
Why do you need these changes? Also, if you check if a token is a truthful value, why do you use `token ?? ''`?

Reply:
In the backend, the auth.jwt.plugin.ts checks the following:

const authHeader = request.headers[HttpHeader.AUTHORIZATION];

        if (!authHeader) {
            throw new HttpError({
                message: ErrorMessage.MISSING_TOKEN,
                status: HttpCode.UNAUTHORIZED,
            });
        }
        
But without that condition I added, the header arrives at the backend with the word Bearer, so the backend validation does not return MISSING_TOKEN because authHeader equals Bearer even if there is no token..

Reply:
Yes, I get it, but still, why do you need this additional ?? check?

Reply:
Without this, when a request is sent without a token, the backend response is INVALID_TOKEN: 'Token is no longer valid. Please log in again.' instead of MISSING_TOKEN: 'You are not logged in.'

Reply:
not fixed!

Reply:
```js
'' == false;
if (token) { // token == true
    headers.append(
        HttpHeader.AUTHORIZATION,
        `Bearer ${token}`,
    );
}
```